lib/tpm2_options.c

```diff
@@ -190,8 +191,14 @@ void tpm2_print_usage(const char *command, struct tpm2_options *tool_opts) {
             } else {
                 printf(" ");
             }
-            printf("[ -%c | --%s%s]", opt->val, opt->name,
-                    opt->has_arg ? "=<value>" : "");
+            if (isalpha(opt->val)) {
```

Comment:
Thank you for fixing this, I noticed the other day it was printing numbers like 1, 2, etc, now that non-ascii short options are mapped into the option structure.

Reply:
Fixed in updated PR.